File: csharp-netstandard/src/Cloud.Governance.Client/Model/WorkspaceStatus.cs


using System;
using System.Collections;
using System.Collections.Generic;
using System.Collections.ObjectModel;
using System.Linq;
using System.IO;
using System.Runtime.Serialization;
using System.Text;
using System.Text.RegularExpressions;
using Newtonsoft.Json;
using Newtonsoft.Json.Converters;
using Newtonsoft.Json.Linq;
using System.ComponentModel.DataAnnotations;
using OpenAPIDateConverter = Cloud.Governance.Client.Client.OpenAPIDateConverter;

namespace Cloud.Governance.Client.Model
{
    
    public enum WorkspaceStatus
    {
        Active = 0,

        Deleted = 1,

        Archived = 2,

        Locked = 3,

        TemporaryLocked = 4,

        GroupSiteLocked = 5,

        RestrictAccess = 6

    }

}

File: typescript/src/models/WorkspaceStatus.ts
/* tslint:disable */
/* eslint-disable */
/*
 * Cloud Governance Api
 *
 * Contact: support@avepoint.com
 */

/**
 * 
 * @export
 * @enum {string}
 */
export enum WorkspaceStatus {
    Active = 0,
    Deleted = 1,
    Archived = 2,
    Locked = 3,
    TemporaryLocked = 4,
    GroupSiteLocked = 5,
    RestrictAccess = 6
}

export function WorkspaceStatusFromJSON(json: any): WorkspaceStatus {
    return WorkspaceStatusFromJSONTyped(json, false);
}

export function WorkspaceStatusFromJSONTyped(json: any, ignoreDiscriminator: boolean): WorkspaceStatus {
    return json as WorkspaceStatus;
}

export function WorkspaceStatusToJSON(value?: WorkspaceStatus | null): any {
    return value as any;
}

